v2.1/configure-replication-zones.md

```diff
@@ -105,425 +95,58 @@ Constraints can be specified such that they apply to all replicas in a zone or s
-
-# Remove the replication zone for a database:
-$ cockroach zone rm <database> <flags>
-
-# Remove the replication zone for a table:
-$ cockroach zone rm <database.table> <flags>
-
-# Remove the replication zone for an index:
-$ cockroach zone rm <database.table@index> <flags>
-
-# Remove the replication zone for a table or index partition:
-$ cockroach zone set <database.table.partition> --file=<zone-content.yaml> <flags>
-
-# View help:
-$ cockroach zone --help
-$ cockroach zone ls --help
-$ cockroach zone get --help
-$ cockroach zone set --help
-$ cockroach zone rm --help
-~~~
-
-## Flags
-
-The `zone` command and subcommands support the following [general-use](#general) and [logging](#logging) flags.
-
-### General
-
-Flag | Description
------|------------
-`--disable-replication` | Disable replication in the zone by setting the zone's replica count to 1. This is equivalent to setting `num_replicas: 1`.
-`--echo-sql` | Reveal the SQL statements sent implicitly by the command-line utility. For a demonstration, see the [example](#reveal-the-sql-statements-sent-implicitly-by-the-command-line-utility) below.
-`--file`<br>`-f` | The path to the [YAML file](#replication-zone-format) defining the zone configuration. To pass the zone configuration via the standard input, set this flag to `-`.<br><br>This flag is relevant only for the `set` subcommand.
-
-### Client connection
-
-{% include {{ page.version.version }}/sql/connection-parameters.md %}
+## View replication zones
 
-See [Client Connection Parameters](connection-parameters.html) for more details.
+Use the [`SHOW ZONE CONFIGURATIONS`](show-zone-configurations.html) statement to view details about existing replication zones.
 
-Currently, only members of the `admin` role can configure replication zones. By default, the `root` user belongs to the `admin` role.
-
-### Logging
+## Manage replication zones
 
-By default, the `zone` command logs errors to `stderr`.
-
-If you need to troubleshoot this command's behavior, you can change its [logging behavior](debug-and-error-logs.html).
+Use the [`CONFIGURE ZONE`](configure-zone.html) statement to add, modify, reset, and remove replication zones.
 
 ## Basic examples
 
-These examples focus on the basic approach and syntax for working with zone configuration. For examples demonstrating how to use constraints, see [Scenario-based Examples](#scenario-based-examples).
+These examples focus on the basic approach and syntax for working with zone configuration. For examples demonstrating how to use constraints, see [Scenario-based examples](#scenario-based-examples).
 
-###  List the pre-configured replication zones
+More examples are also on the [`CONFIGURE ZONE`](configure-zone.html) and [`SHOW ZONE CONFIGURATIONS`](show-zone-configurations.html) pages.
 
-Newly created CockroachDB clusters start with some special pre-configured replication zones:
+### View all replication zones
```

Comment:
This is where I would link out to the reference doc with e.g. "For more information, see SHOW ZONE CONFIGURATIONS".